fetch_teleop/scripts/tuck_arm.py

```diff
@@ -73,8 +76,10 @@ def __init__(self):
     def run(self):
         move_thread = None
         if not is_moveit_running():
-            rospy.loginfo("starting moveit")
+            rospy.loginfo("Starting MoveIt...")
             move_thread = MoveItThread()
+        else:
+            prospy.loginfo("MoveIt already started...")
```

Comment:
Pretty sure this is supposed to be "rospy" not "prospy"

Reply:
🤦 thanks!